implementation/service_discovery/src/service_discovery_impl.cpp

```diff
@@ -2332,7 +2334,7 @@ service_discovery_impl::handle_eventgroup_subscription(
                 << std::setw(4) << _instance << "."
                 << std::setw(4) << _eventgroup << "]"
                 << " not valid: Event configuration ("
-                << (std::uint32_t)_info->get_reliability()
+                << +static_cast<std::uint32_t>(_info->get_reliability())
```

Comment:
Remove typo.

```suggestion
                << static_cast<std::uint32_t>(_info->get_reliability())
```

Reply:
Not a typo, see https://github.com/COVESA/vsomeip/pull/560#discussion_r1550589301